WooCommerce/Classes/ViewRelated/Orders/Order Details/Address Edit/EditOrderAddressFormViewModel.swift

```diff
@@ -11,7 +12,11 @@ final class EditOrderAddressFormViewModel: AddressFormViewModel, AddressFormView
 
     /// Order to be edited.
     ///
-    private let order: Yosemite.Order
+    private var order: Yosemite.Order {
+        didSet {
+            syncFieldsWithOrder()
```

Comment:
As in the case of customer notes, we have to update the fields when the order is updated